doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -274,8 +274,8 @@ This chapter describes the system parameters used by the components of the X-Roa
 | server-conf-service-cache-size                   | 1000                                       |   |   | Maximum number of services to keep cached |
 | server-conf-acl-cache-size                       | 100000                                     |   |   | Maximum number of access rights to keep cached in memory. |
 | enforce-client-is-cert-validity-period-check     | false                                      |   |   | Whether to reject a request when client information system certificate is expired or not yet valid. |
-| backup-encrypted                                 | false                                      |   |   | Whether the security server configuration backups are encrypted. |
-| backup-public-key-path                           | /etc/xroad/backupkeys                      |   |   | Directory where additional backup encryption keys are stored. |
+| backup-encryption-enabled                        | false                                      |   |   | Whether to encrypt security server backup files using server's OpenPGP key. |
+| backup-encryption-keyids                         |                                            |   |   | Comma-separated list of additional recipient OpenPGP key identifiers. |
```

Comment:
The document version number has not been incremented

Reply:
Intentional, this is a fix to earlier documentation change. I think it is sufficient to log the change just once since the documentation is not yet part of a released version. A reader is interested what overall changed between versions 6.26.0 and 7.0; a detailed change log is anyway in the git history).